vita3k/modules/module_parent.cpp

```diff
@@ -168,8 +168,7 @@ void call_import(EmuEnvState &emuenv, CPUState &cpu, uint32_t nid, SceUID thread
         const std::unordered_set<uint32_t> lle_nid_blacklist = {};
         log_import_call('L', nid, thread_id, lle_nid_blacklist, pc);
         write_pc(cpu, export_pc);
-        // TODO: invalidate cache for all threads. Now invalidate_jit_cache is not thread safe.
-        invalidate_jit_cache(cpu, pc, 4 * 3);
+        emuenv.kernel.invalidate_jit_cache(pc, 4 * 3);
```

Comment:
Isn't this function not thread safe?

Reply:
Sorry, I forgot to revert it. I was trying to make this function thread safe but was not successful. Now fixed.